nntrainer/src/databuffer_func.cpp

```diff
@@ -204,7 +204,7 @@ void DataBufferFromCallback::updateData(BufferType type) {
     testReadyFlag = DATA_NOT_READY;
     if (buf_size - (*cur_size) > 0) {
       /** @todo Update to support multiple inputs later */
-      status = callback(vec_arr, veclabel_arr, &endflag);
+      status = callback(vec_arr, veclabel_arr, &endflag, NULL);
```

Comment:
Is this okay not to pass userdata in this context?

Reply:
Currently, we don't have user data.
This will be added to the properties and then passed once data buffer is refactored.